templates/repo/issue/fields/textarea.tmpl

```diff
@@ -1,6 +1,22 @@
-<div class="field">
+<div class="field markdown-textarea">
 	{{template "repo/issue/fields/header" .}}
 	{{/* FIXME: preview markdown result */}}
-	{{/* FIXME: required validation for markdown editor */}}
-	<textarea name="form-field-{{.item.ID}}" placeholder="{{.item.Attributes.placeholder}}" {{if and .item.Validations.required .item.Attributes.render}}required{{end}}>{{.item.Attributes.value}}</textarea>
+	<textarea class="textarea" name="form-field-{{.item.ID}}" placeholder="{{.item.Attributes.placeholder}}" {{if and .item.Validations.required .item.Attributes.render}}required{{end}}>{{.item.Attributes.value}}</textarea>
+
+	{{$ := .Context.Data}}
+	{{template "shared/combomarkdowneditor" (dict
+		"locale" $.locale
+		"MarkdownPreviewUrl" (print $.Repository.Link "/markup")
+		"MarkdownPreviewContext" $.RepoLink
+		"TextareaName" "content"
+		"TextareaContent" .item.Attributes.value
+		"TextareaPlaceholder"  .item.Attributes.placeholder
+		"DropzoneParentContainer" "form, .ui.form"
+	)}}
 </div>
+
+{{if $.IsAttachmentEnabled}}
+<div class="field markdown-textarea">
```

Comment:
If you have 2 "textarea" elements, then there are 2 dronzone components?

Reply:
Ah, Yes. Maybe one dronzone is enough for one issue. 
But it seems that we have a different design to GitHub.
GitHub can upload files as blob files.
![image](https://user-images.githubusercontent.com/18380374/235425179-3bc3ab64-99d7-42b1-bd03-92952028904b.png)

Reply:
I see. We use dropzone.js to handle file upload process.
I think each combo markdown editor needs one dropzone, and it looks good if the file link can be automatically added to the editor after uploaded.